tests/suites/deploy/deploy_charms.sh

```diff
@@ -48,7 +48,7 @@ run_deploy_lxd_profile_charm_container() {
 
     ensure "test-deploy-lxd-profile-container" "${file}"
 
-    juju deploy cs:~juju-qa/bionic/lxd-profile-without-devices-5 --to lxd
+    juju deploy cs:~juju-qa/bionic/lxd-profile-without-devices-5 --to lxd --series=bionic
```

Comment:
Doesn't the charm URL force bionic anyway? Or is the `--series=bionic` added for consistency with the changes below?

Reply:
Latter.